src/Components/NewTodo.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Fragment, useState } from "react";
 import { useDispatch } from "react-redux";
-import { addTodo } from "services/actions/addTodo";
+import { addTodo } from "services/actions/actions";
```

Comment:
Isn't that suppose to be similar? Please look after that. Thanks!
```suggestion
import { actions } from "services/actions/actions";
```